doc/designs/lms.md

```diff
@@ -111,15 +116,29 @@ __owur int EVP_DigestVerifyHBSInit(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
                                    const char *props, EVP_PKEY *pkey,
                                    const OSSL_PARAM params[],
                                    const unsigned char *sig,
-                                   size_t siglen);
-   
+                                   size_t siglen);   
 /* Alias: since the function signature is identical */
 #define EVP_DigestVerifyHBSUpdate(a,b,c) EVP_DigestVerifyUpdate(a,b,c)
 
 __owur int EVP_DigestVerifyHBSFinal(EVP_MD_CTX *ctx);
-
 ```
 
+Proposal 2:
+
+Add a Init_ex function and leave the Final the same.
+
+i.e.
+EVP_DigestVerifyInit_ex3(..., const unsigned char *sig, size_t siglen);
```

Comment:
@t8m mentioned `_with_signature` for the libcrypto\<->provider interface.  Could that be a good idea here as well, instead of yet another `ex`?